bids/layout/models.py

```diff
@@ -377,15 +377,17 @@ class BIDSImageFile(BIDSFile):
         'polymorphic_identity': 'image_file'
     }
 
-    def get_image(self):
+    def get_image(self, **kwargs):
         """Return the associated image file (if it exists) as a NiBabel object
+
+        Any keyword arguments are passed to ``nibabel.load``.
         """
         try:
             import nibabel as nb
-            return nb.load(self.path)
-        except Exception:
+            return nb.load(self.path, **kwargs)
+        except Exception as e:
```

Comment:
out of curiosity, why the import inside the try?

Reply:
nibabel used to be an optional dependency, I think.